bot/cogs/WikiListener.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+
+import aiohttp
+
+import json
+
+import ast
+
+import pprint
+
+import re
+
+
+"""
+
+Handle requests and grab and returns detail from wikipedia using MediaWiki.
+
+"""
+
+
+async def get_json(url):
```

Comment:
Why not json.loads inside this function so that you don't have to repeat code in future usages of this function?

Reply:
Done.